packages/gnome-shell/src/ui/popupMenu.d.ts

```diff
@@ -72,37 +72,41 @@ declare class PopupBaseMenuItem extends St.BoxLayout {
 
 /**
  * @see https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/ui/popupMenu.js#L285
- * @version 46
+ * @version 47
  */
 export namespace PopupMenuItem {
     export interface ConstructorProps extends PopupBaseMenuItem.ConstructorProps {}
 }
 
 /**
  * @see https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/ui/popupMenu.js#L286
- * @version 46
+ * @version 47
  */
 export class PopupMenuItem extends PopupBaseMenuItem {
     constructor(text: string, params?: Partial<PopupMenuItem.ConstructorProps>);
+    /** @hidden Defined only to resolve type conflicts */
+    override _init(config?: PopupMenuItem.ConstructorProps): void;
```

Comment:
Does `@hidden` have any effect? I can't find that tag in the JSDoc reference. https://jsdoc.app/#block-tags

Reply:
Don't know. I just copied that from similar overrides, didn't actually check whether it exists 🤔

honestly I don't know where that comes from 🤔

Perhaps @Totto16 knows more?

Reply:
Mhh the jsdoc equivalent is @ignore: https://typedoc.org/tags/hidden/

Reply:
But since we're typescript wouldn't we use typedoc anyway? 🤔

Reply:
> But since we're typescript wouldn't we use typedoc anyway? 🤔

Oh , good point. I wasn't aware that there is separate specification of JSDoc for TypeScript. Well, nevermind then.